src/app/components/list/list.service.js

```diff
@@ -41,16 +37,24 @@
 
     // Cache a list for future offline use
     List.prototype.cache = function () {
+      var lfusers = $localForage.instance('users');
+      var lflists = $localForage.instance('lists');
       var request = { limit: 50, offset: 0, sort: 'name'};
+      lflists.setItem(this._id, this);
```

Comment:
It could still fail to set the item if there isn't enough space though and cause an error